iree/build_defs.oss.bzl

```diff
@@ -86,6 +85,3 @@ def iree_flatbuffer_cc_library(**kwargs):
         gen_reflections = False,
         **kwargs
     )
-
-def iree_glsl_vulkan(**kwargs):
```

Comment:
This should have a corresponding change to the upstream `build_defs` file. I'll import manually and test that.